### DISCUSSION TITLE
2.8 vs 2.8.4

### DISCUSSION THREAD

**Discussion by espressotech** [artisan-roaster-scope/artisan#1264] [Q&A] [answered]
in v2.8 when I hit the On button I get temperature data on readings than to start record I hit start
but on v2.8.4 if I hit On it start running and recording automatically what is the change?
I prefer to chose when to start the recording  so I downgraded to 2.8 but can I use on and start separately on 2.8.4 and future versions?
thank you

**Comment by MAKOMO** [comment]
The behavior you describe is very strange. Could you please attach your `artisan-settings.aset` here. For this, export your settings via menu Help >> Save Settings, rename the file to end with .txt, like in `artisan-settings.txt` and drag and drop this file hier on the GitHub Web page to a comment filed for uploading (attach the file to a reply by email will not work!). I will take a look at your setup and try to duplicate your findings. Thanks for your support!

**Comment by espressotech** [comment, marked as answer]
(https://www.loom.com/share/86a0dc5a7bae42ffb519d28ab5f30d33?sid=0dd01c3f-f61a-4013-9424-0e464d498554)
[artisan-settings 2.8.4.txt](https://github.com/artisan-roaster-scope/artisan/files/12683303/artisan-settings.2.8.4.txt)
I recorded also the behavior

**Reply by MAKOMO** [reply]
Well, you told Artisan to do so.

<img width="1072" alt="Screenshot 2023-09-21 at 12 36 39" src="https://github.com/artisan-roaster-scope/artisan/assets/13909011/6b2f5893-001e-4e5a-a0d6-249287c3ce6c">

**Reply by espressotech** [reply]
obviously I did not put it there. 
yes I noticed that when I compared 2 versions and looked how it is on 2.8 and it is the same but it does not happen,
how are the 2 cells should look like blank? attached the v2.8 setting
[artisan-settings 2.8.txt](https://github.com/artisan-roaster-scope/artisan/files/12684108/artisan-settings.2.8.txt)

**Comment by MAKOMO** [comment]
> obviously I did not put it there

I did not put it either;)

> yes I noticed that when I compared 2 versions and looked how it is on 2.8 and it is the same but it does not happen, 

All Artisan installations on your computer share the same settings. However, the `button(START)` command was introduced after Artisan v2.8.0.

> how are the 2 cells should look like blank? 

Both empty

**Reply by espressotech** [reply]
thank you
